lib/geordi/commands/dump.rb

```diff
@@ -5,18 +5,35 @@
 `geordi dump -l tmp/staging.dump` (with the `--load` option) sources the
 specified dump file into the development database.
 
+
+If you are using multiple databases per environment, pass the database name like this:
+
+    geordi dump -d primary
+
+Loading a dump into one of multiple local databases is not supported yet.
+
+When called with the `--load` option, sources the specified dump into the
+development database.
+
```

Comment:
I assume these lines got mislocated during a rebase. Please move them to the very end of the description (right before L24) and drop L15+16, which seems like a rebase relict as well.